internal/i18n/i18n.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/nicksnyder/go-i18n/v2/i18n"
 	"github.com/zitadel/logging"
 	"golang.org/x/text/language"
-	"sigs.k8s.io/yaml"
+	"sigs.k8s.io/yaml" // TODO: not gopkg.in/yaml.v3?
```

Comment:
?

Reply:
Isit intended to use the k8s yaml library?